couchexport/transforms.py

```diff
@@ -15,7 +15,7 @@ def couch_to_excel_datetime(val, doc):
         dt_val = None
         for fmt in COUCH_FORMATS:
             try:
-                val = datetime.datetime.strptime(val, fmt)
+                dt_val = datetime.datetime.strptime(val, fmt)
```

Comment:
Oops, sorry about this! Definitely happened either in my refactor or in last-minute 'fix' to accept both time formats (w/ and w/o usecs). Thanks for finding